src/main/java/team/gif/robot/Robot.java

```diff
@@ -13,12 +13,7 @@
 import team.gif.robot.commands.drivetrain.DriveTank;
 import team.gif.robot.commands.arm.ArmManualControl;
 import team.gif.robot.commands.elevator.ElevatorManualControl;
-import team.gif.robot.subsystems.Arm;
-import team.gif.robot.subsystems.Collector;
-import team.gif.robot.subsystems.CollectorPneumatics;
-import team.gif.robot.subsystems.Drivetrain;
-import team.gif.robot.subsystems.Elevator;
-import team.gif.robot.subsystems.SwerveDrivetrain;
+import team.gif.robot.subsystems.*;
```

Comment:
You have a setting wrong in your project file. We never use  wildcard for importing. 

In Intelli-J: File | Settings | Editor | Code Style | Java | Imports
set Class count to use import with *: 500
set Names count to use static import with *: 300

Reply:
These still need to be put back.
```suggestion
import team.gif.robot.subsystems.Arm;
import team.gif.robot.subsystems.Collector;
import team.gif.robot.subsystems.CollectorPneumatics;
import team.gif.robot.subsystems.Drivetrain;
import team.gif.robot.subsystems.Elevator;
import team.gif.robot.subsystems.SwerveDrivetrain;
```